README.md

```diff
@@ -10,26 +10,7 @@ composer require answear/messenger-heartbeat-bundle
 `Answear\MessengerHeartbeatBundle\AnswearMessengerHeartbeatBundle::class => ['all' => true],`  
 should be added automatically to your `config/bundles.php` file by Symfony Flex.
 
-## Setup
-
-Add middleware as the last element to prevent duplicate messages when RabbitMQ server closes a channel with PRECONDITION_FAILED 
-https://www.rabbitmq.com/consumers.html#acknowledgement-timeout
-
-```yaml
-# config/packages/messenger.yaml
-framework:
-    messenger:
-        default_bus: messenger.bus.default
-        buses:
-            messenger.bus.default:
-                middleware:
-                    ...
-                    - 'Answear\MessengerHeartbeatBundle\Middleware\TransportExceptionsMiddleware'
```

Comment:
nie łapię już tam tego wyjątku HeartbeatConnectionLostException, ale sam wyjątek zostawiam, może być dedykowany

Reply:
To był dupochron przed dublującymi się wiadomościami jak poleciał błąd ze strony rabbita, ciekawe co teraz będzie się działo.